execution/executor-benchmark/src/lib.rs

```diff
@@ -195,11 +197,12 @@ pub fn run_benchmark<V>(
 
     let mut overall_measuring = OverallMeasuring::start();
 
-    if let Some(transaction_generators) = transaction_generators {
+    if let Some((transaction_generators, phase)) = transaction_generators {
         generator.run_workload(
             block_size,
             num_blocks,
             transaction_generators,
+            phase,
```

Comment:
As `phase` is always `Arc::new(0)`, do we have to output it from `init_workload` function? Why not just use `Arc::new(0)` here instead?